openapi/index.yaml

```diff
@@ -541,6 +541,42 @@ paths:
           description: Cannot regenerate service key.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/organizations/{organization_fiscal_code}/logo":
+    parameters:
+      - name: organization_fiscal_code
+        in: path
+        type: string
+        required: true
+        description: The Organization Fiscal Code.
+    put:
+      summary: Upload organization logo.
+      description: |
+        Upsert a logo for an Organization.
+        This API operation needs a valid API key
+        otherwise 403 forbidden will be returned to the caller.
+      operationId: uploadOrganizationLogo
+      parameters:
+        - name: body
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/Logo"
+          description: A base64 string representation of the organization logo PNG image.
+      responses:
+        "200":
+          description: Logo uploaded.
```

Comment:
Do we prefer/can return 201 along with the reference to the just uploaded image?

Reply:
The io-functions-admin endpoint, that is called inside this endpoint on io-functions-services' side, returns 201 with the link to the uploaded image. I don't remember well, but in this case we are avoiding to expose infos about storage and CDN links to external subjects. cc: @gunzip